.github/workflows/release.yml

```diff
@@ -56,6 +59,17 @@ jobs:
           echo "VERSION=$version" >> $GITHUB_ENV
           echo "PRIOR_VERSION=$prior_version" >> $GITHUB_ENV
 
+      - name: Install Cosign
+        uses: sigstore/cosign-installer@v3.4.0
+
+      - name: Sign the artifacts with GitHub OIDC Token
+        run: |
+          mkdir sigstore
+          ls **/build/libs/*.jar | xargs -I % cosign sign-blob --yes % --bundle %.sigstore
+          mv **/build/libs/*.sigstore ./sigstore/
```

Comment:
Do you have an example of the results? I wonder what would be the difference between this and the Gradle plugin for sigstore. Have you considered that one instead?